modules/administration/pages/ansible-setup-control-node.adoc

```diff
@@ -3,20 +3,22 @@
 
 To set up an Ansible control node, execute the following steps from the {productname} {webui}.
 
-.Procedure: Setting up Ansible Control Node
+[NOTE]
+====
+To configure a {productname} client the Ansible software must be available for operating system of the client.
+This usually means Ansible must come from the client's software distribution.
+For example, Ansible is available for {sle} 15 SP6 or SP7.
+====
 
-. In the {productname} {webui}, navigate to menu:Admin[Setup Wizard > Products], verify that [literal]``SUSE Linux Enterprise Server 15 SP5 x86_64`` and [literal]``{productname} Client Tools for SLE 15 x86_64`` are selected and synchronized.
```

Comment:
Shouldn't we mention that the old ansible version can be found in SLE15 client tools? We are dropping all the information about this in the page

Reply:
We have new set of SLE15 client tools in 5.1 and those client tools don't have ansible. 

For 5.0, yes, we need to keep the info about the client tools.

Reply:
Hmmm, I still see `ansible` at:
- https://build.suse.de/package/show/Devel:Galaxy:Manager:Head:MLMTools-Beta-SLE15/ansible
- https://build.suse.de/package/show/Devel:Galaxy:Manager:5.1:MLMTools-SLE15/ansible

Aren't these the new client tools? Maybe the package are present even if not really submitted.. we would need some clarifications.

Reply:
I'll merge this one now.  If needed we must create a follow-up PR.

Reply:
@meaksh sorry missed your comment before. I don't know why ansible package is displayed in IBS but I don't see that here https://updates.suse.de/SUSE/Products/MultiLinuxManagerTools/SLE-15/x86_64/product/ and it isn't expected. I will check with @deneb-alpha